pkg/clustermanager/cluster_manager.go

```diff
@@ -523,6 +523,22 @@ func (c *ClusterManager) getWorkloadClusterKubeconfig(ctx context.Context, clust
 	return nil
 }
 
+// CreateEKSAReleaseBundle applies the eks-a release bundle to the cluster.
+func (c *ClusterManager) CreateEKSAReleaseBundle(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec) error {
```

Comment:
can this be done from the eks-a installer?

Reply:
I ended up using this wrapper because while creating the cluster's namespace we use the cluster Client that is a private Retrier Client field used by the cluster manager. I won't be able to use the clusterctl client if I were to move this code.

Reply:
why do you need "the clusterctl client"? we should only need a k8s client wrapped by a retrier
which the installer should already have?

Reply:
My bad, I meant the cluster client.